src/interfaces/events.ts

```diff
@@ -0,0 +1,11 @@
+export interface Event {
+    title: string,
+    description: string,
+    links: string[],
```

Comment:
I recommend having only one link to redirect to the selected event page, and then in the event-name.json file, include all the links you want to display.

```suggestion
    link: string,
```

Reply:
At `src/interfaces/communities` we got a similar behaviour. Would you recommend to implement this change in that source file, too?

Reply:
Thank you, guys 🐥